imports/plugins/included/payments-paypal/client/templates/settings/express.js

```diff
@@ -17,16 +18,12 @@ Template.paypalExpressSettings.helpers({
 });
 
 AutoForm.hooks({
-  "paypal-update-form": {
+  "paypal-update-form-express": {
     onSuccess: function () {
-      Alerts.removeSeen();
-      return Alerts.add("Paypal settings saved.", "success", {
-        autoHide: true
-      });
+      return Alerts.toast(i18next.t("admin.settings.saveSuccess"), "success");
     },
     onError: function (operation, error) {
-      Alerts.removeSeen();
-      return Alerts.add("Paypal settings update failed. " + error, "danger");
```

Comment:
I am not sure how much I care about it but I think that the `Alerts.removeSeen` was preventing this:
![reaction](https://user-images.githubusercontent.com/72819/31154158-c043d63e-a8d7-11e7-9867-1459c37b20fc.png)

Was there a particular reason you removed it?